clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -703,18 +703,43 @@ public FetchManagerMetrics(Metrics metrics, String metricGrpPrefix) {
         }
 
         public void recordTopicFetchMetrics(String topic, int bytes, int records) {
+            Map<String, String> metricTags = new HashMap<>();
+            metricTags.put("topic", topic);
```

Comment:
I noticed that we prefix the node-id tag values with "node-" for the connection metrics in Selector. I think this makes it a little easier to navigate if you're using something like jconsole. Would it make sense to do the same here?